app/src/main/res/values/strings.xml

```diff
@@ -816,7 +816,7 @@
   <string name="nhid_error_cannot_assign_to_patient_with_existing_ok">OK</string>
 
   <!-- Select ongoing hypertension treatment dialog -->
-  <string name="select_ongoing_hypertension_treatment_title">Did patient take BP medicines any time in the last month?</string>
+  <string name="select_ongoing_hypertension_treatment_title">Confirm if patient took BP medicines anytime in the last month</string>
```

Comment:
This error dialog is not showing the entire message after we made this change. The expected behaviour is to show the entire message ([figma design ref](https://www.figma.com/file/vmZlmlFSCOrEYzTMkqsNB8/Simple-Production?node-id=17674%3A285)) Also the UI of this dialog is off. Can you PTAL?

Reply:
Yep, that's resolved in a later PR, it requires us to change a style.

Reply:
>Also the UI of this dialog is off. Can you PTAL?

WDYM? The UI seems the same as the designs 🤔

Reply:
So, we can ignore this change for now. I have talked to Mahima about a small issue with the current dialog, we will be changing them, will open a different PR for it.